crud.lua

```diff
@@ -23,6 +23,11 @@ local stats = require('crud.stats')
 
 local crud = {}
 
+-- @refer crud.version
+-- @tfield string _VERSION
+--  Contains hardcoded value of the CRUD version.
```

Comment:
Let's now write about implementation here

Reply:
Updated

Reply:
Well, sorry, it was a typo. I meant to write "let's not write". Let's not write about "hardcoded" in any public API description. And, well, I think it's better to remove the details about its maintenance too. Sorry

Reply:
Let's use just `module version`

Reply:
Done